local-harnesses/build.config.json.template

```diff
@@ -9,7 +9,6 @@
     "log_path": "bhapi.log",
     "enable_startup_wait_period": false,
     "datapipe_interval": 1,
-    "graph_driver": "neo4j",
```

Comment:
This is incorrect, we want BHCE to continue to dev with neo4j by default. We need to remove the second "graph_driver": "pg" instance